x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_table.tsx

```diff
@@ -287,16 +287,16 @@ export const ActionsLogTable = memo<ActionsLogTableProps>(
             </EuiScreenReaderOnly>
           ),
           render: (actionListDataItem: ActionListApiResponse['data'][number]) => {
+            const actionId = actionListDataItem.id;
             return (
               <EuiButtonIcon
                 data-test-subj={getTestId('expand-button')}
-                onClick={onClickCallback(actionListDataItem)}
+                // triggers the expansion
+                onClick={onClickCallback(actionId)}
```

Comment:
I think the problem is here, since the callbacks are cached between renders, the `actionListDataItem` is stale, even though the component does re-render and update the component's `items` prop.

After changing this callback to use the action id instead, so we can find and use that action from the passed-in prop using the id, it kind of works on the second toggle open. Still investigating...